Error code: 0xdb1b4a
Error retrieving block: 0xd4ee1e, HTTPSConnectionPool(host='svc.blockdaemon.com', port=443): Max retries exceeded with url: /ethereum/mainnet/native (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1121f1960>: Failed to resolve 'svc.blockdaemon.com' ([Errno 8] nodename nor servname provided, or not known)"))
Error retrieving block: 0xd7e200, HTTPSConnectionPool(host='svc.blockdaemon.com', port=443): Max retries exceeded with url: /ethereum/mainnet/native (Caused by SSLError(SSLError(1, '[SSL: TLSV1_ALERT_DECODE_ERROR] tlsv1 alert decode error (_ssl.c:1007)')))
